teleport-node/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Stage1: build from source
+FROM quay.io/cybozu/golang:1.15-focal AS build
+
+ARG TELEPORT_VERSION=6.0.0
+RUN git clone --depth 1 --branch v${TELEPORT_VERSION} https://github.com/gravitational/teleport && \
+    cd teleport && \
+    make build/teleport OS=linux
+
+# Stage2: setup runtime container
+FROM quay.io/cybozu/ubuntu-debug:20.04
+
+COPY --from=build /work/teleport/build/teleport /usr/local/bin/
+COPY update-necocli.sh /etc/profile.d/
+
+RUN adduser --disabled-password --gecos "" --uid 10000 cybozu
+RUN chown cybozu:cybozu /home/cybozu
```

Comment:
It makes no effect because the home dir is overwritten by k8s volume.

Reply:
If we add `VOLUME /home/cybozu`, this makes sense because the new volume
will be initialized with the volume contents including the owner and permission bits.